thavalon-webapp/src/components/Home.tsx

```diff
@@ -3,7 +3,6 @@ import React from 'react';
 export function Home(): JSX.Element {
     return (
         <div>
-            <h1>Hello World!</h1>
```

Comment:
should this at least say THavalon or something?

Reply:
We could make the "Join Game" modal the landing page, since that's what people are looking for most of the time

Reply:
That's not a bad idea, but I think it is harder than just making the join game modal come up, due to the routing stuff. I changed it to welcome to Thavalon for now.